.prettierrc

```diff
@@ -3,6 +3,5 @@
   "singleQuote": true,
   "trailingComma": "all",
   "printWidth": 120,
-  "proseWrap": "never",
-  "endOfLine": "lf"
+  "proseWrap": "never"
```

Comment:
since prettier v2.0.0, "lf" is the default value. So the problem is not relevant to `endOfLine` option.